docs/developer-guide/tasks/README.md

```diff
@@ -148,6 +148,39 @@ tasks:
     <a class="btn btn-primary" href="/plugins/core/tasks/flows/io.kestra.core.tasks.flows.Switch">Switch Task documentation</a>
 </div>
 
+### If
+
+This task processes a set of tasks conditionally depending on a condition. 
+The condition must coerce to a boolean. Boolean coercion allows 0, -0, null and '' to coerce to false,  all other values to coerce to true.
```

Comment:
Is it possible for a value to be undefined?

Reply:
It's Java so undefined didn't exist and will be null